presto-hive/src/main/java/com/facebook/presto/hive/s3/PrestoS3FileSystem.java

```diff
@@ -237,6 +240,8 @@ public void initialize(URI uri, Configuration conf)
         String userAgentPrefix = conf.get(S3_USER_AGENT_PREFIX, defaults.getS3UserAgentPrefix());
         this.skipGlacierObjects = conf.getBoolean(S3_SKIP_GLACIER_OBJECTS, defaults.isSkipGlacierObjects());
         this.s3StorageClass = conf.getEnum(S3_STORAGE_CLASS, defaults.getS3StorageClass());
+        this.webIdentityEnabled = conf.getBoolean("presto.hive.s3.web.identity.auth.enabled", false);
+        this.webIdentityTokenFile = conf.get("presto.hive.s3.web-identity-token-file");
```

Comment:
nit: please use the static variable defined in `S3ConfigurationUpdater` for config key.